packages/ui/src/components/DateInput/index.tsx

```diff
@@ -256,18 +260,13 @@ export const DateInput = ({
   readOnly = false,
   tooltip,
   showMonthYearPicker,
+  selectsMultiple,
   'data-testid': dataTestId,
 }: DateInputProps) => {
   const uniqueId = useId()
   const localId = id ?? uniqueId
 
-  // Linked to: https://github.com/Hacker0x01/react-datepicker/issues/3834
-  const ReactDatePicker =
-    (DatePicker as unknown as { default: typeof DatePicker }).default ??
-    DatePicker
-
-  const localeCode =
-    (typeof locale === 'string' ? locale : locale?.code) ?? 'en-GB'
+  const localeCode = (typeof locale === 'string' ? locale : locale) ?? 'en-GB'
```

Comment:
we need to check why they remove the code of locale?.code in the type.

Reply:
https://github.com/Hacker0x01/react-datepicker/blob/f735e60e97da8e5f4066276da9cf601e85e9eeed/src/date_utils.ts#L64